tests/node_tests/events/events.js

```diff
@@ -16,7 +16,9 @@ emitter.emit({});                         // err: `event` must be a string
 emitter.eventNames().pop();               // ok: returns string[]
 emitter.eventNames('foo')                 // err: does not process args
 
-emitter.listeners('foo').pop()();         // ok: returns Function[]
+emitter.listeners('foo').pop();           // ok: returns Function[]
+emitter.listeners('foo').pop()();         // err: Function | void
+emitter.listeners('foo').pop()?.();       // ok: optional chaining of Function | void
```

Comment:
Added a new call to just `pop` to show it's ok and that it's ok with chaining. The existing test case is now an error (but sandwiched between new tests).

Most of the changes in .exp are line numbers being off :(